### PR TITLE
Some tweaks

### PR DESCRIPTION
**Pull request by pluma**
Created the other PR a bit too early. This one doesn't change any behaviour but adds a note to the README and follows your naming convention more closely.

### CODE REVIEW

**Comment by ksmithut** [comment]
perfect, thanks :smile:

**Comment by ksmithut** [comment]
Alright, that's been published to npm as version 3.5.0

**Comment by pluma** [comment]
Awesome. Thanks :smiley:

File: README.md [hunk 1]
```diff
@@ -105,6 +105,12 @@ module.exports = function (grunt) {
 };
 ```
 
+or via the CLI
+
+```sh
+$ bower-ls js css
```
Comment (shinnn) [inline README.md]: bower-files

Comment (ksmithut) [inline README.md]: I though of that, too, but the cli really only lists the components, so `ls` seems like a good convention to keep with. Maybe an alias for `bower-files` would be good to put in. Thoughts?

Comment (shinnn) [inline README.md]: Sounds good.

Comment (pluma) [inline README.md]: Dang. I could have sworn I checked that. Another reason not to write PRs late at night, I guess.

**Pull request merged**